tests/modules/test_tf_embedding.py

```diff
@@ -0,0 +1,44 @@
+from box_embeddings.parameterizations.tf_box_tensor import (
+    TFBoxTensor,
+    TFBoxFactory,
+)
+from box_embeddings.modules.tf_embedding import TFBoxEmbedding
+import tensorflow as tf
+
+
+def test_simple_creation() -> None:
+    embedding = TFBoxEmbedding(5, 10)
+    assert embedding.num_embeddings == 5
+    assert (
+        embedding.embedding_dim
+        == 10 * embedding.box_factory.box_subclass.w2z_ratio
+    )
+    assert embedding.box_factory.name == 'mindelta_from_vector'
+
+
+def test_creation_with_box_factory() -> None:
+    box_factory = TFBoxFactory('sigmoid_from_vector')
+    embedding = TFBoxEmbedding(5, 10, box_factory=box_factory)
+    assert embedding.box_factory.name == 'sigmoid_from_vector'
+
+
+def test_forward() -> None:
+    embedding = TFBoxEmbedding(5, 10)
+    box_factory = TFBoxFactory('mindelta_from_vector')
+    W = embedding.embeddings
+    assert W.shape == (5, 20)
+    box_tensor = box_factory(W)
+    box_tensor_direct = box_tensor.data.numpy()[2:4, ...]
+    inputs = tf.Variable([2, 3])
+    box_tensor_from_emb = embedding.call(inputs)
```

Comment:
Don't use `call` explicitly. Use the class instance as a callable `box_tensor_from_emb = embedding(inputs)`

Reply:
Noted. Will make this change. The function signature `call()` in embedding will be replaced by `__call__()` with same functionality